code/game/objects/items/trash.dm

```diff
@@ -131,6 +131,24 @@
 /obj/item/trash/plate
 	name = "plate"
 	icon_state = "plate"
+	var/clean = FALSE
+/obj/item/trash/plate/clean
+	icon_state = "cleanplate"
+	desc = "Clean enough to eat on, probably."
```

Comment:
Could dynamically change the desc depending on whether or not it's clean or not.